### PR TITLE
[FIX] web: reslove ribbon overlaps while open dropdown menu

### PR DESCRIPTION
**Pull request by sabr-odoo**
Specification:
The ribbon overlaps with the dropdown menu.

image

Expected behavior:
The ribbon should not overlap with the dropdown menu.

Task-3940927

### CODE REVIEW

File: addons/web/static/src/views/widgets/ribbon/ribbon.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
     height: var(--Ribbon-wrapper-height);
     overflow: hidden;
     position: absolute;
+    z-index: 0;
```
Comment (Polymorphe57) [inline addons/web/static/src/views/widgets/ribbon/ribbon.scss:9]: I don't think it is appropriate. You should check what is wrong with --Ribbon-z-index

**Comment by Polymorphe57** [comment]
What is the purpose of having that variable then?

**Comment by sabr-odoo** [comment]
Hello @Polymorphe57 ,

changes have been made. Could you please take a look at it?

Thank you

**Comment by Polymorphe57** [comment]
@robodoo r+

**Pull request closed without merging**